walkthroughs/week-4-tdd/intro/src/main/java/com/google/sps/Greeter.java

```diff
@@ -1,27 +0,0 @@
-// Copyright 2019 Google LLC
```

Comment:
Is this file supposed to be deleted?

Reply:
No, I did not mean to delete it. I was using git to move around what I did and did not want to commit, and I accidentally deleted that file. I'm not sure how to get it back.